Add `ngrok` usage to README (#3260)
bosagora/agora: devel/testnet/README.md

that `boa1xrwuel4csj4acdfdr5c6xufewa7r5l5g83cp5uax2lyxaakkhwc27aghk7m.validators.testnet.bosagora.io`
points to your own IP address.

## Exposing your validator on the internet with `ngrok`
As you have read in the previous section, you can run a validator, but if the validator runs
on the localhost, so you cannot reach it from another validaors. So you can use the [ngrok](https://ngrok.com)
to run a validator on your local machine. If you only run a node, not a validator, please skip

> this section.

Reviewer: ```suggestion
As you have read in the previous section, you can run a validator, but if the validator runs
on a local computer without a publicly exposed ip, it cannot be reached from other validators. One option is to use a service like [ngrok](https://ngrok.com) to expose a local port to the public. If you only run a fullnode, not a validator, please skip this section.
```

You should stop your validator before following the following instructions.

**First**, to use `ngrok`, you need to register yourself on the `ngrok` web site and create an account.
`ngrok` has a free plan that allows you to use the software for free with the limination of only

> one preocess at a time with a maximum of four tunnels and a random port, but it is enough to

Reviewer: ```suggestion
`ngrok` has a free plan that allows you to use the software for free with the limitation of only
one process at a time with a maximum of four tunnels and a random port, but it is enough to
```

test your validator.

**Second**, once you download the `ngrok` client, you have to connect your account to the client
(you need to do this operation once only) using the following command:
```
./ngrok authtoken <your_auth_token>
```
You can find your authentication token in the Auth section of your accout dashboard.
![Getting ngrok authtoken](ngrok_auth.png)

> **Third**, to expose your validator using `ngrok`, simply expose the local port using the `tcp` command.

Reviewer: ```suggestion
**Third**, to expose your validator using `ngrok`, simply expose a local port, currently not being used, using the `tcp` command.
```
